projects/plugins/crm/eslint.config.mjs

```diff
@@ -0,0 +1,14 @@
+import makeBaseConfig from 'jetpack-js-tools/eslintrc/base.mjs';
+
+export default [
+	...makeBaseConfig( import.meta.url, {
+		react: true,
+		textdomain: 'zero-bs-crm',
+		envs: [ 'browser', 'jquery' ],
+	} ),
+	{
+		rules: {
+			'jsdoc/require-jsdoc': 'off',
```

Comment:
Interesting...when I was doing JS linting for CRM in #40249 it was requiring JSDoc.

Reply:
Trunk still has this rule in `projects/plugins/crm/.eslintrc.js`. 🤷
https://github.com/Automattic/jetpack/blob/3c012a818223d53c22c27d1f124bcee68b46981e/projects/plugins/crm/.eslintrc.js#L11

Possibly what you ran into is that disabling this rule stops it from requiring new jsdoc to be added where there isn't one, but if there is one already then all the other rules demand all the different parts?